src/ml-service/packages/sensor_location/app/modules/locate_model.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+from pymongo import MongoClient
+import json
+
+def connect_mongo():
+    '''
+    Connects to local MongoDB
+    '''
+    #client = MongoClient('mongodb+srv://lillian:fosho@cluster0-99jha.gcp.mongodb.net/test?retryWrites=true&w=majority')
+    client = MongoClient("mongodb://localhost:27017")
```

Comment:
When we starting refactoring our codebase, these connection details of our databases will easily fall under` AirQo-api/src/locate/config/db_connection`.
This will ensure that we always call our database connection once throughout our codebase.